operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -91,6 +91,16 @@ if [ -f ${STATEFILE} ] && [ `grep -c "FAILED_NOT_RESTARTABLE" ${STATEFILE}` -eq
   exit $RETVAL
 fi
 
+if [ -x ${LIVENESS_PROBE_CUSTOM_SCRIPT} ]; then
+  $LIVENESS_PROBE_CUSTOM_SCRIPT
+elif [ -O ${LIVENESS_PROBE_CUSTOM_SCRIPT} ]; then
+  chmod 770 $LIVENESS_PROBE_CUSTOM_SCRIPT && $LIVENESS_PROBE_CUSTOM_SCRIPT
+fi
+if [ $? != 0 ]; then
+  trace SEVERE "Execution of custom script ${LIVENESS_PROBE_CUSTOM_SCRIPT} failed."
```

Comment:
Execution of custom liveness probe script '${LIVENESS_PROBE_CUSTOM_SCRIPT}' failed.